easybuild/easyconfigs/g/GROMACS/GROMACS-2023.2-cpeGNU-22.12-hipSYCL-GPU.eb

```diff
@@ -0,0 +1,116 @@
+# Based on recipe from Rasmus Kronberg (CSC) 
+#
+easyblock = 'CMakeMake'
+
+name =          'GROMACS'
+version =       '2023.2'
+versionsuffix = '-hipSYCL-GPU' 
+
+homepage = 'http://www.gromacs.org'
+
+whatis = [
+    'Description: GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.',
+]
+
+description = """
+GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
+Newtonian equations of motion for systems with hundreds to millions of
+particles.
+
+Binaries included:
+* gmx_mpi: Compiled with MPI and hipSYCL (GPU) support
```

Comment:
I suggest clarifying how this differs from the above two, i.e. that this will be an MPI build hence multi-node capable and will use VkFFT but without HeFFTe hence PME can not be run on multiple GPUs.